client/src/workers/create-transaction-worker-script.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  Transaction,
+  TransactionInstruction,
+  PublicKey,
+  Account,
+} from "@solana/web3.js";
+import * as Bytes from "utils/bytes";
+import { CreateTransactionMessage } from "./create-transaction-rpc";
+
+const self: any = globalThis;
```

Comment:
In a web worker, self is the global context object. TypeScript wasn't dealing well with this for some reason, but I was able to set it to globalThis.